src/app/components/hmcts-global-header/hmcts-global-header.component.ts

```diff
@@ -11,9 +11,10 @@ export class HmctsGlobalHeaderComponent {
     @Input() set userLoggedIn(value) {
         this.userValue = value;
     }
-    @Input() headerTitle: { name: string; url: string };
+    @Input() headerTitle: { name: string; url: string; hideBranding: boolean};
     @Input() navigation;
     @Input() isBrandedHeader: boolean;
+    @Input() hideBranding: boolean;
```

Comment:
I don't think this one is needed as you're using the one from the `headerTitle`, above.